test/api/v01/output_test.rb

```diff
@@ -302,11 +302,12 @@ def test_returned_keys_json_result
     }]
 
     methods[:periodic_ortools][:problem][:configuration][:first_solution_strategy] = nil
+    dimensions = [:time, :distance]
```

Comment:
this adds an extra line but is only used once ?

Reply:
This variable is used within every step of the each loop. + It complies with rubocop warns